src/views/PurchaseOrderDetail.vue

```diff
@@ -84,6 +85,8 @@ import {
 import { defineComponent } from 'vue';
 import { add, saveOutline, timeOutline } from 'ionicons/icons';
 import ReceivingHistoryModal from '@/views/ReceivingHistoryModal.vue'
+import { useStore } from 'vuex';
+import { mapGetters } from "vuex";
```

Comment:
Remove this import and use a single import when importing from the same package.

Reply:
Okay, Sir.